apps/crn-server/src/data-providers/contentful/calendars.data-provider.ts

```diff
@@ -57,8 +57,9 @@ export class CalendarContentfulDataProvider implements CalendarDataProvider {
       skip: 0,
       order: [CalendarsOrder.NameAsc],
       where: {
-        ...(maxExpiration ? { expirationDate_lt: maxExpiration } : {}),
-        ...(resourceId ? { resourceId } : {}),
+        ...(maxExpiration || resourceId
+          ? { googleApiMetadata_exists: true }
```

Comment:
It's not possible to filter data inside objects directly  https://www.contentful.com/developers/docs/references/graphql/#/reference/collection-filters/limitations